packages/sequence/sequence.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+homepage: "https://github.com/c-cube/sequence/"
+bug-reports: "https://github.com/c-cube/sequence/issues"
+license: "BSD-2-clauses"
+doc: "https://c-cube.github.io/sequence/"
+tags: ["sequence" "iterator" "iter" "fold"]
+dev-repo: "https://github.com/c-cube/sequence.git"
+build: [make "build"]
+install: ["jbuilder" "install"]
+build-test: [make "test"]
+build-doc: [make "doc"]
+remove: ["ocamlfind" "remove" "sequence"]
```

Comment:
There's no need for install/remove stanzas if you're using jbuilder. Jbuilder generates .install files which make this all obsolete.

Also, I would suggest you replace your build an build-test commands with something like:

```
build: [
  ["jbuilder" "build" "-p" name "-j" jobs]
]
build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
```

This is better because it doesn't rely on make. Note how we include `"-p" name` in there. This is important if you include more than 1 opam package in your package sources.